### ISSUE TITLE
Shipping Labels: Misalignment on text fields of Address Form

### ISSUE THREAD

**Issue by itsmeichigo** [woocommerce/woocommerce-ios#5084]
**Describe the bug**
It looks like there are some inconsistencies with the text field alignments:

| Current | Expected |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/135253752-94711332-2d33-4843-84b1-a1e59d45734f.jpeg" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/135253802-58129b89-6754-44e4-88ac-fef45972200f.jpeg" width=320 />

**Comment by pmusolino** [comment]
What do you think if we align those text fields to the right as we do in other sections of the app (like when adding a new customer)? @adamzelinski @joe-keenan

**Comment by ghost** [comment]
@pmusolino I think we should align them as per designs thanks. As we've got a lot of forms in shipping labels etc using that style.

**Comment by joe-keenan** [comment]
I agree with Adam. It’s slightly easier to read a left-aligned set of values as a complete set (like an address) than it is to read a right-aligned one. 
Also, thank you for addressing this. It has been bugging me every time I look at it!

**Comment by joshheald** [comment]
Automated triage note: this was tested and commented by Codex using device/simulator automation.

Attempted on iPad Pro 11-inch (M4) simulator running iOS 18.6. I could not reach the shipping-label address form because WDA was unavailable for navigation. Code inspection shows supported stores now default to the revamped WooShipping label flow, while the older address form from this report still exists as a fallback. I did not get live visual evidence for or against the alignment issue in this pass.

I am leaving this GitHub issue open and applying the `triage: automation attempted` label for follow-up visibility.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by JorgeMucientes** [comment]
Closing this because the mobile shipping labels flow was fully revamped in 2025, and this issue predates that revamp / targets legacy shipping labels behavior. If this still reproduces in the current shipping labels flow, please open a new issue with updated steps and screenshots.
